app/page.tsx

```diff
@@ -1,19 +1,13 @@
-import LandingPageMainHero from './components/main-content/LandingPageMainHero';
 import LinktaLogoWithText from './components/layout/LinktaLogoWithText';
-import Footer from './components/layout/Footer';
+import LandingPageMainHero from './components/main-content/LandingPageMainHero';
 
 export default function Home() {
   return (
     <div>
-      <header>
-        <LinktaLogoWithText />
-      </header>
-      <main>
+        <div className="sm:hidden ml-0">
+          <LinktaLogoWithText />
```

Comment:
Kinski mentioned this in another PR - this div doesn't seem to be actually be rendering at any size. I'm not exactly sure why, but we already have the logo rendering in the `header` in `layout.tsx`. Can it be removed from this file or is there a need here?

Reply:
I tested with this removed. There is no Linkta logo on the homepage when <= 390px width

Reply:
Ohhh I see. It looks like that behavior is coming from `globals.css`:

```
  @media screen and (max-width: 390px) {
    body > footer, body > header {
      @apply hidden;
    }
  ...other styling
  }
```

Removing `body > header` from this rule will fix that issue as well. Of course, it would also display anything else in our header at that screen size. Currently the logo is the only child element of the header. What do you think?

Reply:
Alternatively, if we want to keep this div to show the logo on the home page only (and not the sign up page) on mobile, we'll want to update the breakpoint. Because `sm` is used here, the element is hidden at 390px (iphone 12 width) but visible below.